docs/test-api-matrix.md

```diff
@@ -7,7 +7,7 @@ application|:white_check_mark: partially|:heavy_check_mark:||
 bucket||||partially within application
 dependency|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
 file|:heavy_check_mark:|:heavy_check_mark:||
-import||||
+import|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
```

Comment:
The binding part is not covered for import endpoint, which is OK, but don't add :heavy_check_mark: in the first column, leave it empty or add minus symbol or something similar.

Reply:
I am leaving it empty